src/scenes/ProgressReports/Detail/WorkflowEventsHistory.tsx

```diff
@@ -0,0 +1,76 @@
+import { East } from '@mui/icons-material';
+import { Box, Collapse, Divider, Skeleton, Typography } from '@mui/material';
+import { Fragment } from 'react';
+import { ProgressReportStatusLabels as StatusLabels } from '~/api/schema/enumLists';
+import { FormattedDate } from '~/components/Formatters';
+import { Redacted } from '~/components/Redacted';
+import { RichTextView } from '~/components/RichText';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+
+interface WorkFlowEventHistoryProps {
+  report?: ProgressReportDetailFragment;
+  checked?: boolean;
+}
+
+export const WorkFlowEventHistory = ({
```

Comment:
Maybe calling this `WorkflowEventList` would help distinguish it from the "history" page.
Be sure the file & prop names match the component name too.